src/masonite/validation/Validator.py

```diff
@@ -125,6 +124,31 @@ def message(self, attribute):
     def negated_message(self, attribute):
         return '{} must not be a valid timezone'.format(attribute)
 
+class one_of(BaseValidation):
```

Comment:
Where are you using this?  and it does not seem you have added in `self.register()` in `ValidationFactory `